peer.go

```diff
@@ -2037,6 +2037,20 @@ func (p *peer) reenableActiveChannels() {
 	}
 }
 
+// getLocalDeliveryScript returns an script to close the channel out to. If
+// an upfront shutdown script was set, it is returned. Otherwise, a random
```

Comment:
random is probably not the right wording here... Perhaps "a new delivery script is returned"?